ui-admin/src/status/StatusPage.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCheckCircle, faExclamationCircle } from '@fortawesome/free-solid-svg-icons'
+import Api from 'api/api'
+import LoadingSpinner from 'util/LoadingSpinner'
+
+export const StatusPage = () => {
+  const [operational, setOperational] = useState<boolean>()
+
+  const loadSystemStatus = async () => {
+    const systemStatus = await Api.getSystemStatus()
+    setOperational(systemStatus.ok)
+  }
+
+  useEffect(() => {
+    loadSystemStatus()
+  }, [])
+
+  return <div className="d-flex flex-column align-items-center justify-content-center vh-100">
+    {operational === undefined ?
+      <LoadingSpinner /> :
+      <SystemStatus operational={operational}/>
+    }
+    <div className="text-center">
+      <div className="text-muted">
+        Please contact <a href="mailto:support@juniper.terra.bio">support@juniper.terra.bio</a> for additional support.
```

Comment:
outside the scope of this PR, but we should have this email pulled from a single file.  Maybe the zendeskUtils file, and rename that to supportUtils